### PR TITLE
Adding GitHub build aciton

### PR DESCRIPTION
**Pull request by automaton82**
Hey @perepujal , this PR updates the Github CI action and adds a regular Android CI plus a Fastlane build. You can see the results of them working against this branch successfully here:

* https://github.com/tux4kids/Tuxpaint-Android/actions/runs/6859174835/job/18651082824
* https://github.com/tux4kids/Tuxpaint-Android/actions/runs/6859174832/job/18651082828

I've re-pointed the PR at the `master` branch so it's ready to merge. However, if we decide to bring it into `master` then you should follow the [advice documented here](https://discuss.circleci.com/t/remove-circleci-from-github-repo/1144) and remove the current CircleCI from the webhook so it no longer runs (unless you know why it always fails and think this PR is a bad idea).

I don't have access to the `Settings` tab of this repo so I cannot do it.

Let me know what you think.

### CODE REVIEW

**Comment by perepujal** [comment]
Hi, could you test if now you can access the Settings tab so you can apply the changes yourself?

**Comment by automaton82** [comment]
I have the `Settings` tab now, but it's missing most of the stuff that is usually there in my other repos, including the webhooks section.

**Review by perepujal — COMMENTED** [review]
Hi, I added a comment about the license of gradle-wrapper.jar but shows as "Pending" so I don't know if you saw it.
I repeat here, sorry if you see twice.
Which license it is?
Are we allowed to redistribute it?

**Comment by perepujal** [comment]
Hopefully you should be able to got to webhooks section now.

File: gradle/wrapper/gradle-wrapper.jar
[the remark this answers is not in the document]
Comment (automaton82) [reply gradle/wrapper/gradle-wrapper.jar:1]: Yes your comment showed up now :).

The jar file is produced during gradle build, you can read more about it here:
https://docs.gradle.org/current/userguide/gradle_wrapper.html

> To make the Wrapper files available to other developers and execution environments, you need to check them into version control. Wrapper files, including the JAR file, are small. Adding the JAR file to version control is expected. Some organizations do not allow projects to submit binary files to version control, and there is no workaround available.

Gradle is covered by Apache 2.0 and we are allowed to distribute:

https://docs.gradle.org/current/userguide/licenses.html

Gradle themselves distribute it too:

https://github.com/gradle/gradle/blob/54c01e9bb06e1e3b1fdba58d7cbd0b940e97d641/gradle/wrapper/gradle-wrapper.jar

Hopefully this answers your question!

**Comment by automaton82** [comment]
> Hopefully you should be able to got to webhooks section now.

Yes I do, thank you!

File: gradle/wrapper/gradle-wrapper.jar
[the remark this answers is not in the document]
Comment (perepujal) [reply gradle/wrapper/gradle-wrapper.jar:1]: Thanks for the onfo, then it is OK to include the .jar file.

**Pull request merged**